views/_products.erb

```diff
@@ -50,6 +47,10 @@
       <% end %>
     </tbody>
   </table>
+
+  <div style="padding-left: 5px;">
+    <%= paginate products, param_name: :products_page %>
```

Comment:
I should be able to pass `params: {tab: 'products'}` here and receive this param server side. Unfortunately its not working though and I can't track down why. Looks like its not being written into the link so it must be something with the helper